airflow/dags/resources/stages/parse/table_definitions/aave/StableDebtToken_event_Approval.json

```diff
@@ -0,0 +1,54 @@
+{
+    "parser": {
+        "abi": {
+            "anonymous": false,
+            "inputs": [
+                {
+                    "indexed": true,
+                    "internalType": "address",
+                    "name": "owner",
+                    "type": "address"
+                },
+                {
+                    "indexed": true,
+                    "internalType": "address",
+                    "name": "spender",
+                    "type": "address"
+                },
+                {
+                    "indexed": false,
+                    "internalType": "uint256",
+                    "name": "value",
+                    "type": "uint256"
+                }
+            ],
+            "name": "Approval",
+            "type": "event"
+        },
+        "contract_address": "SELECT stableDebtToken FROM ref(`PoolConfigurator_v3_event_ReserveInitialized`) GROUP BY stableDebtToken",
```

Comment:
refs should use a quote instead of a backtick, i.e. `ref('PoolConfigurator_v3_event_ReserveInitialized')` instead of `ref(``PoolConfigurator_v3_event_ReserveInitialized``)`